lib/workers/dependency.rb

```diff
@@ -59,6 +58,7 @@ def generate_stats(locations)
         results[:outdated] = type unless results[:outdated] || num == 0
       end
     end
+    results[:outdated] = :unknown if results[:outdated_total] == 0 && results[:ok] == 0
     results[:outdated] = :up_to_date unless results[:outdated]
```

Comment:
Can you talk me through the logic here?

Reply:
If we don't manage to fetch any dependencies, all the categories will be empty. I thought it would be weird if that yielded an "all dependencies ok". So: set the status to `:unknown` if there are neither any outdated dependencies, nor any ok dependencies (but possibly unknown dependencies).